tmt-app/src/Components/Home.js

```diff
@@ -1,63 +1,112 @@
 import React from "react";
-import '../Styling/Home.css'
+import '../Styling/Home.css';
+import {PrimaryBtn, FlexContainer} from '../Styling/CustomStyling.js';
+import {Link, Container, Typography} from '@mui/material';
 import ForwardArrow from '../Images/Forward.png';
 import MilkTeaBlob from '../Images/MilkTeaBlob.png';
 import LycheeRectangle from '../Images/LycheeRectangle.png';
 import MilkTeaSwiggle2 from '../Images/MilkTeaSwiggle2.png';
 import ThaiTeaSwiggle1 from '../Images/ThaiTeaSwiggle1.png';
 import ArrowRightAltIcon from '@mui/icons-material/ArrowRightAlt';
 
+
+
 function Home(){
     return(
-        <div className="Home_Container">
-
-            <div className="Home_Path_Container">
+        <Container 
+            sx={{
+                minWidth:"100%", 
+                paddingTop:"7.5rem", 
+                overflow:"hidden", 
+                minHeight:"100vh"
+            }}
+        >
+            <Container>
                 <div className="Home_Path_Wrapper">
-                    <span>Student Projects</span>
-                    <img src={ForwardArrow} alt="ForwardArrow"></img>
-                    <span>Major Picker</span>
+                    <span>
+                        <Typography variant="CustomSubHeading" color="#616587">Student Projects</Typography>
+                    </span>
+                    <img src={ForwardArrow} alt="ForwardArrow"/>
+                    <span>
+                        <Typography variant="CustomSubHeading" color="#616587">Major Picker</Typography>
```

Comment:
the `color` attribute here will probably be best styled via a style sheet or MUI theme.

Not a deal breaker leaving this in, but I think we'll most likely delete this later when we start implementing more of the theme at a global level